docs/swagger.yaml

```
@@ -26,4 +49,23 @@ paths:
       summary: Insert new test struct
       tags:
       - test
+  /test/{uuid}:
+    delete:
+      consumes:
+      - application/json
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 8 but found 6

(indentation)